packages/themes/src/g90.js

```diff
@@ -113,7 +112,7 @@ export const disabled01 = gray80;
 export const disabled02 = gray70;
 export const disabled03 = gray50;
 
-export const highlight = blue90;
+export const highlight = blue70;
```

Comment:
Looks like the test is failing because we're referencing `blue70` here but not importing it